### PR TITLE
fabric-sdk: add keep alive config support #279

### PR DESCRIPTION
**Pull request by adecaro**
Signed-off-by: Angelo De Caro <adc@zurich.ibm.com>

### CODE REVIEW

File: platform/fabric/core/generic/config/config.go [hunk 1]
```diff
@@ -87,6 +87,26 @@ func (c *Config) TLSRootCertFile() string {
...
+	return c.configService.GetDuration("fabric." + c.prefix + "tls.keepalive.server.interval")
+}
+
+func (c *Config) KeepAliveServerTimeout() time.Duration {
+	return c.configService.GetDuration("fabric." + c.prefix + "tls.keepalive.server.timeout")
+}
+
+func (c *Config) KeepAliveServerMinInterval() time.Duration {
+	return c.configService.GetDuration("fabric." + c.prefix + "tls.keepalive.server.minInterval")
+}
```
Comment (mbrandenburger) [inline platform/fabric/core/generic/config/config.go:108]: What would happen if the configuration does not contain the entry? Would `c.configService.GetDuration` just panic or do we have default values somewhere? Maybe directly from `core_template.go`?

File: integration/nwo/fabric/topology/core_template.go [hunk 2]
```diff
@@ -282,6 +282,14 @@ fabric:
...
         - {{ .PeerLocalTLSDir Peer }}/ca.crt
       rootCertFile: {{ .CACertsBundlePath }}
+      keepalive:
+       client:
+         interval: 60s
+         timeout: 600s
+       server:
+         interval: 60s
+         timeout: 600s
+         minInterval: 60s
```
Comment (mbrandenburger) [inline integration/nwo/fabric/topology/core_template.go:292]: Just thinking ... how would a FSC user be able to change these settings without touch `core_template.go`?

Comment (adecaro) [reply integration/nwo/fabric/topology/core_template.go:292]: just changing the configuration file, no?

File: platform/fabric/core/generic/config/config.go (hunk 1 above)
Comment (adecaro) [reply platform/fabric/core/generic/config/config.go:108]: good point, let's test :)

File: integration/nwo/fabric/topology/core_template.go (hunk 2 above)
Comment (adecaro) [reply integration/nwo/fabric/topology/core_template.go:292]: or we need explicit support on the topology. Let's wait for a first request on this 😅

File: platform/fabric/core/generic/config/config.go (hunk 1 above)
Comment (adecaro) [reply platform/fabric/core/generic/config/config.go:108]: actually, it just return the default value the type of variable can get. Duration gets 0, being an int64.

**Pull request merged**